include/proxsuite/proxqp/dense/utils.hpp

```diff
@@ -343,11 +343,15 @@ template<typename T>
 void
 global_dual_residual(Results<T>& qpresults,
                      Workspace<T>& qpwork,
+                     const Model<T>& qpmodel,
                      preconditioner::RuizEquilibration<T>& ruiz,
                      T& dual_feasibility_lhs,
                      T& dual_feasibility_rhs_0,
                      T& dual_feasibility_rhs_1,
-                     T& dual_feasibility_rhs_3)
+                     T& dual_feasibility_rhs_3,
+                     T& rhs_duality_gap,
+                     T& duality_gap,
+                     T sqrt_max_dim)
```

Comment:
Consider evaluating it within the function. This is a minor quantity to compute.

Reply:
OK, I make the change